src/web/services/course.service.ts

```diff
@@ -254,6 +254,6 @@ export class CourseService {
     const paramsMap: Record<string, string> = {
```

Comment:
The method should be in the student service right?

Reply:
I just replaced the endpoint, it was already being called from course service, do you want me to move it to student service? It should be fine since the only component using this method already has a dependency on student service

Reply:
We have to do that as it is not the responsibility of course service to gent students. Remember SRP?